articles/libraries/_includes/_error_responses.md

```diff
@@ -0,0 +1,35 @@
+### Error Responses
+
+The error response returned for the `/usernamepassword/login` endpoint is different from that retured from the `/co/authenticate` endpoint.
```

Comment:
I might alter this to just say "errors returned in previous versions were different than those which are returned in the newest version" rather than naming endpoints, since we've done an OK job of not needing to talk about low level items in the rest of the guides.